docs/changes/wcs/11549.feature.rst

```diff
@@ -0,0 +1,3 @@
+Updated bundled version of WCSLIB to version 7.6. See
```

Comment:
Don't usually updating extern lib end up in "other" section, not "feature"?

Reply:
I don't think we need what's new entry, but if others disagree, please put a "what's new needed" label from the existing labels.

Reply:
For astropy 4.0.2 , the entry was in the "new feature" section. That is why I put in in the "feature" section